apps/public-docsite-v9/src/Concepts/Migration/GettingStarted.stories.mdx

```diff
@@ -0,0 +1,148 @@
+
+## Major changes and why we made them
+
+### Build-time styles
+
+In previous versions, styles were created and applied at runtime. Components used a theme object on React.Context to create styles.
+
+In v9, styles are created at build time to define class names. Components apply these class names at runtime based on state. A theme object is used to set CSS variables in the DOM which component styles reference.
+
+[Griffel](https://griffel.js.org/), a CSS-in-JS framework, provides build-time transforms, atomic CSS, and type-safe styles for v9.
+
+Build-time styles significantly improve rendering performance and can be optimized to reduce bundle size.
+
+### Design tokens
+
+In previous versions, the theme defined properties specific to component, component part, and state. For example, `inputTextHovered` or `buttonBorderDisabled`.
+
+In v9, the theme properties are defined from the design tokens names. Design tokens are specific to neutral/brand, usage, and state. For example, `colorNeutralForeground2Hover`, `colorBrandForegroundLinkPressed`, or `spacingVerticalXL`.
+
+The Fluent 2 design language provides a practical number of tokens useful to all components without degrading performance. As new components are added, new tokens are not required. This makes each theme less fragile and allows components to depend on token values being defined.
+
+### TSX/JSX children
+
+In previous versions, components had props that accepted data objects or arrays. It was the component's responsibility to render each child based on the data. Components often exposed callbacks so callers could override rendering children.
+
+For example, v8's `ContextualMenuButton` `menuProps` prop would be passed a hierarchy of menu items to render.
+
+In v9, most components support declaring the children as TSX/JSX elements. Component children are responsible for rendering themselves.
+
+For example, `MenuList` can have `MenuItem`, `MenuDivider`, `MenuItemRadio`, etc. as children.
+
+Declarative children provide idiomatic React development.
+
+- Children can be full-fledged components supporting data and event binding and access context.
+- Components don't have to provide render callbacks as child components render themselves.
+- Components can be extended with custom child types.
+- Components don't have to define additional types to express hierarchy.
+- The caller controls React performance tuning techniques like memoization and lazy loading.
+
+### Slots
+
+In previous versions, components would expose render callback props. These allowed callers to customize how a component part (e.g. header, footer, item, content) were rendered.
+
+In v9, components define slots that allow callers to replace a component part with their own. Slot props are like the previous callbacks but more powerful; they support literals, JSX/TSX, or render functions.
```

Comment:
The section title uses TSX/JSX so let's stick with that everywhere. Or maybe just shorten it to "JSX"? We write everything as TSX but always talk about "JSX".

```suggestion
In v9, components define slots that allow callers to replace a component part with their own. Slot props are like the previous callbacks but more powerful; they support literals, TSX/JSX, or render functions.
```